### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by IgorBalog-Eng**
Fix for TransferContract in WSS ArtifactRequestMessage flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +54,102 @@ public ResponseEntity<?> proxyRequest(@RequestHeader HttpHeaders httpHeaders,
...
+else if (requestHeader instanceof ArtifactRequestMessage && isBigPayload(((ArtifactRequestMessage) requestHeader).getRequestedArtifact().toString())) {
+	return encodePayload == true ? encodePayload(BigPayload.BIG_PAYLOAD.getBytes()) : BigPayload.BIG_PAYLOAD;
+	}
+	return  encodePayload == true ? encodePayload(createResponsePayload().getBytes()) : createResponsePayload();
+			
+```
+Current example has 2 payloads, one small - json representing "John Doe" and other - big payload, that has several hundred lines of text. For your use case, you can simplify it and just handle *if (requestHeader instanceof ArtifactRequestMessage)* use case. 
+
+What is needed is to modify
+**private String createResponsePayload() ** method and provide logic that will fit your needs. You can add here part to read from some DB, call API, read file from filesystem, anything you need.
```
Comment (davidjovanovic) [inline README.md:104]: the text is not bold, delete last space before **

File: README.md [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+ * Clone [WebSocket Message Streamer](https://github.com/Engineering-Research-and-Development/true-connector-websocket_message_streamer)
+ * Once this project is cloned, run mvn clean install
+
+This will install 2 internal libraries that are needed by DataApp project.
+
+After that you can run mvn clean package in the root of the dataApp project, to build it.
+
+---
+
+Other solution is to use provided libraries on GitHub Package. To do so, you will have to modify Apache Maven seetings.xml file like following:
```
Comment (davidjovanovic) [inline README.md:23]: typo seetings.xml

File: README.md [hunk 3]
```diff
@@ -1,7 +1,49 @@
...
 [![License: AGPL](https://img.shields.io/github/license/Engineering-Research-and-Development/true-connector-basic_data_app.svg)](https://opensource.org/licenses/AGPL-3.0)
 
-## Dedicated endpoint in dataApp
+## Building dataApp
+
+**Requirement:**
+
+ `Java11` `Apache Maven`
+ 
+To build dataApp you will have to do one of the following:
```
Comment (davidjovanovic) [inline README.md:10]: Does this only reffer to the bulletin or bulletin and GH Packages solution?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
 
+## [0.1.12-SNAPSHOT] - 2022-11-04
+
+### Added
+
+ - developer documentation for modifying DataApp, consumer and provider vise
+ 
+### Changed
+
+ - fixed transferCOntract field for WSS ArtifactRequestMessage flow
```
Comment (davidjovanovic) [inline CHANGELOG.md:12]: typo transferCOntract

File: README.md (hunk 2 above)
Comment (IgorBalog-Eng) [reply README.md:23]: fixed

File: README.md (hunk 1 above)
Comment (IgorBalog-Eng) [reply README.md:104]: done

File: README.md (hunk 3 above)
Comment (IgorBalog-Eng) [reply README.md:10]: Added Solution 1 and Solution 2, to make difference between those 2

File: CHANGELOG.md (hunk 4 above)
Comment (IgorBalog-Eng) [reply CHANGELOG.md:12]: fixed

**Review by davidjovanovic — APPROVED** [review]

**Pull request merged**